src/main/java/com/powsybl/openloadflow/network/impl/LfBusImpl.java

```diff
@@ -33,8 +33,12 @@
 
     private final double nominalV;
 
+    private boolean voltageControlCapacility = false;
+
     private boolean voltageControl = false;
 
+    private int voltageControlSwitchOffCount = 0;
```

Comment:
I am not sure about the necessity of having "Off" in the variable name. Maybe ````voltageControlSwitchingCount````could be better ?

Reply:
Ok but in this case I have to increment when voltageControl is switched on and off and adapt the test for PV/PQ/PV max switch (we will have double value)

Reply:
Ah okay, so no leave it like that, I understand know. Maybe just a small comment in the code ?

Reply:
Comment added in the Javadoc